### PR TITLE
Updates for v2.8 in basic tutorial lessons and setup

### PR DESCRIPTION
**Pull request by rswamina**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes (https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/280) and (https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/275)

### CODE REVIEW

**Review by ehogan — CHANGES_REQUESTED** [review]
Thanks @rswamina! All the points in the issues referenced on this PR have been addressed, and I have built the site locally and can confirm the changes you made render ok :)

I know this has nothing to do with your changes, but I noticed an empty point in the quick start guide that I believe to be caused by the `- {: .callout}` on L36; would it be possible for you to remove this, please?

This PR handles updates to the basic tutorial, so episodes beyond and including "Writing your own recipe" have not been updated, as expected. However, the introduction episode has not been updated; is this expected?

File: _episodes/04-recipe.md [hunk 1]
```diff
@@ -335,14 +357,16 @@ Do you recognize the basic recipe structure that was introduced in episode 1?
...
 > > - **timeseries/tas_global**: creates `/preproc/timeseries/tas_global`, which
-> >   contains preprocessed data for each of the input datasets, and
-> >   `metadata.yml`.
+> >   contains preprocessed data for each of the input datasets, a
+> >   `metadata.yml` file and provenance information in the 
+>>    form of `.xml` files.
 > > - **timeseries/tas_amsterdam**: creates `/preproc/timeseries/tas_amsterdam`,
 > >   which contains preprocessed data for each of the input datasets, plus a
-> >   combined `MultiModelMean`, and `metadata.yml`.
+> >   combined `MultiModelMean`, a`metadata.yml` file and provenance files.
```
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/04-recipe.md:369]: Done.

File: _episodes/02-installation.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Installation"
 teaching: 10
 exercises: 10
-compatibility: ESMValTool v2.6.0
+compatibility: ESMValTool v2.8.0
```
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/02-installation.md:5]: @ehogan - I have made the change.

File: _episodes/04-recipe.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/04-recipe.md]: Done.

File: _episodes/02-installation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/02-installation.md:5]: Done.

File: _episodes/04-recipe.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ We will start by running [examples/recipe_python.yml](https://docs.esmvaltool.
 org/en/latest/recipes/recipe_examples.html)
 
 ```
-esmvaltool run examples/recipe_python.yml
+esmvaltool run recipe_python.yml
```
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/04-recipe.md]: This was a mistake, now fixed. Not sure which line underneath you are referencing?

File: setup.md [hunk 4]
```diff
@@ -235,7 +235,7 @@ local machine and go [here](#github-account-advanced).
 If you are planning on running ESMValTool on your own machine, please make sure
 that you are able to download CMIP data and that you have several GB of space
 available to install conda and ESMValTool, but also enough to make a copy of some
-data (~2Gb).
+data (~125MB).
```
[the remark this answers is not in the document]
Comment (rswamina) [reply setup.md]: @ehogan - Let me know if the changes made work better.

**Comment by rswamina** [comment]
> Thanks @rswamina! All the points in the issues referenced on this PR have been addressed, and I have built the site locally and can confirm the changes you made render ok :)
> 
> I know this has nothing to do with your changes, but I noticed an empty point in the quick start guide that I believe to be caused by the `- {: .callout}` on L36; would it be possible for you to remove this, please?
> 
> This PR handles updates to the basic tutorial, so episodes beyond and including "Writing your own recipe" have not been updated, as expected. However, the introduction episode has not been updated; is this expected?

@ehogan - I did not edit files that were not required for the basic tutorial or files where changes were not needed. I noticed a broken link in the Introduction episode and have fixed that. If anything else needs changing, let me know.

**Comment by rswamina** [comment]
Hi @ehogan  - Thanks a bunch for reviewing the PR so promptly. Let me know if there are pending issues and if not, please approve changes and merge.

File: _episodes/04-recipe.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ehogan) [reply _episodes/04-recipe.md]: Ah, I mis-read the line underneath, it says "user configuration file in your current directory" but I read it as "recipe in your current directory" :)

File: _episodes/02-installation.md [hunk 5]
```diff
@@ -229,10 +229,13 @@ to display the command line help.
...
 > > ~~~
-> > ESMValCore: 2.6.0
-> > ESMValTool: 2.6.0
+> > ESMValCore: 2.8.0
+> > ESMValTool: 2.8.0
 > > ~~~
 > > {: .output}
+> Note that on HPC servers such as JASMIN, sometimes a more recent development 
+> version may be displayed for ESMValTool, for e.g. 
+> `ESMValTool: 2.9.0.dev4+g6948d5512`
```
Comment (ehogan) [inline _episodes/02-installation.md:238]: I would argue that ESMValTool v2.8.0 should be installed on JASMIN, not a development version. I'll raise an ESMValTool issue for this.

File: setup.md [hunk 6]
```diff
@@ -233,11 +233,18 @@ Please skip this section if you are not going to use ESMValTool on your
...
 available to install conda and ESMValTool, but also enough to make a copy of some
-data (~2Gb).
+data (~125MB) needed for this tutorial.
 
-To download the data required by this tutorial, run the following command using
+You can use ESMValTool to automatically download data needed for test recipes. 
+Please see the [Configuration][lesson-configuration] episode 
+or the [configuration file documentation][config-file] for more information.
+This the recommended option as it has the advantage that data is 
+stored in subdirectories, and features  such as wildcards and recording
```
Comment (ehogan) [inline setup.md:244]: There is an extra whitespace between "features" and "such".

**Review by ehogan — APPROVED** [review]
Thanks @rswamina! I noted a few whitespace issues (and something that I will raise with the Tech Lead team via the ESMValTool repository), but I am happy to approve this :)

File: _episodes/02-installation.md (hunk 5 above)
Comment (rswamina) [reply _episodes/02-installation.md:238]: Sure, please tag me as well so I can keep track of updates on that.

**Comment by ehogan** [comment]
I would normally expect the developer to merge the PR; is the standard process for the tutorial for the reviewer to merge? Or did you just want me to help out in this one instance :)

**Comment by rswamina** [comment]
> I would normally expect the developer to merge the PR; is the standard process for the tutorial for the reviewer to merge? Or did you just want me to help out in this one instance :)

I will fix the two white space issues and merge. Thanks!

File: setup.md (hunk 6 above)
Comment (rswamina) [reply setup.md]: Fixed.

File: _episodes/04-recipe.md [hunk 7]
```diff
@@ -335,14 +357,16 @@ Do you recognize the basic recipe structure that was introduced in episode 1?
...
-> >   for each of the input datasets, and a file called `metadata.yml`
-> >   describing the contents of these datasets.
+> >   for each of the input datasets, a file called `metadata.yml`
+> >   describing the contents of these datasets and provenance information in the 
+>>    form of `.xml` files.
 > > - **timeseries/tas_global**: creates `/preproc/timeseries/tas_global`, which
-> >   contains preprocessed data for each of the input datasets, and
-> >   `metadata.yml`.
+> >   contains preprocessed data for each of the input datasets, a
+> >    `metadata.yml` file and provenance information in the
```
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/04-recipe.md]: Fixed :)

File: _episodes/02-installation.md (hunk 5 above)
Comment (ehogan) [reply _episodes/02-installation.md:238]: See https://github.com/ESMValGroup/ESMValTool/issues/3163 :)

**Pull request merged**